packages/soci-snapshotter/soci-snapshotter.spec

```diff
@@ -67,24 +114,49 @@ install -p -m 0755 out/soci-snapshotter-grpc %{buildroot}%{_cross_bindir}
 install -p -m 0755 out/soci %{buildroot}%{_cross_bindir}
 install -p -m 0755 out/fips/soci-snapshotter-grpc %{buildroot}%{_cross_fips_bindir}
 install -p -m 0755 out/fips/soci %{buildroot}%{_cross_fips_bindir}
+install -D -p -m 0644 %{S:100} %{buildroot}%{_cross_unitdir}
 install -D -p -m 0644 %{S:101} %{buildroot}%{_cross_unitdir}
 install -D -p -m 0644 %{S:102} %{buildroot}%{_cross_unitdir}
 
+# Install SOCI config
+install -d %{buildroot}%{_cross_templatedir}
+install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/soci-snapshotter-grpc
+install -p -m 0644 %{S:3} %{buildroot}%{_cross_templatedir}/soci-config-toml
+
 %cross_scan_attribution --clarify %{S:1000} go-vendor vendor
 
+%post igzip -p <lua>
+posix.symlink("%{_cross_bindir}/igzip", "%{_cross_bindir}/soci-gunzip")
+
+%post pigz -p <lua>
+posix.symlink("%{_cross_bindir}/unpigz", "%{_cross_bindir}/soci-gunzip")
+
 %files
 %license LICENSE NOTICE.md
 %{_cross_unitdir}/soci-snapshotter.service
 %{_cross_unitdir}/soci-snapshotter.socket
+%{_cross_unitdir}/etc-soci\\x2dsnapshotter.mount
 %{_cross_attribution_vendor_dir}
 %{_cross_attribution_file}
+%dir %{_cross_factorydir}%{_cross_sysconfdir}/soci-snapshotter-grpc
```

Comment:
We don't seem to be using this factory directory for anything - it would be useful if we wanted to provide a default SOCI config for downstreams that don't have the settings or templates.

Reply:
Gotcha, in that case I will omit it for now.